tests/queries/0_stateless/00626_replace_partition_from_table.sql

```diff
@@ -82,3 +82,49 @@ SELECT count(), sum(d) FROM dst;
 
 DROP TABLE IF EXISTS src;
 DROP TABLE IF EXISTS dst;
+DROP TABLE IF EXISTS rdst;
+
+SELECT 'Temporary tables';
```

Comment:
Let's create a new test for temporary tables and add `no-replicated-database` tag to the new test